src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj

```diff
@@ -3,13 +3,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <PropertyGroup>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <HarvestStablePackage>false</HarvestStablePackage>
   </PropertyGroup>
   <ItemGroup>
+    <HarvestIncludePaths Include="ref/netcore50;runtimes/win8-aot/lib/netcore50" />
+    <HarvestIncludePaths Include="ref/netstandard1.0" />
+    <HarvestIncludePaths Include="ref/netstandard1.2" />
```

Comment:
Right now these are going to apply on desktop too.  That may be OK, but we need to make sure we have an impl there and don't try to apply our corlib-based impl.

Reply:
Can "runtimes/win8/lib/netstandard1.3" the impl for "ref/netstandard1.0" and "ref/netstandard1.2"? or We should have 1 :1 mapping between ref and runtime?

Reply:
There's specifically not a 1:1 mapping.  Ref is more general than runtime.  A ref is typically covered by one or more lib and runtime/rid/lib.  We also typically have a matching lib for ref (since 2.0) and will make that be a "not supported" assembly if we can't provide a default implementation.